templates/logs.html

```diff
@@ -0,0 +1,26 @@
+<!doctype html>
+<html>
+    <head>
+        <link href="../../bootstrap.min.css" rel="stylesheet">
+    </head>
+
+    <body>
+        <div class="container-fluid">
+            <div class="col-md-10 col-md-offset-1">
+                <div class="page-header">
+		    <h2>Container: {{ cid }}</h2>
+                </div>
+	    </div>
+	    <div class="row">
+	        <div class="col-md-6">
+                    <h4>StdOut:</h4>
+		    <p style="white-space: pre-wrap"> {{ out }} </p>
```

Comment:
Also add the class [`pre-scrollable`](http://getbootstrap.com/css/#code-block) to control the vertical length of the output.